common/dynamicconfig/constants.go

```diff
@@ -861,11 +861,11 @@ of Timeout and if no activity is seen even after that the connection is closed.`
 	EnableDeployments = NewNamespaceBoolSetting(
 		"system.enableDeployments",
 		false,
-		`EnableDeployments enables deployments (versioning v3) in all services,
+		`EnableDeployments enables deployments (deprecated versioning v3 pre-release) in all services,
 including deployment-related RPCs in the frontend, deployment entity workflows in the worker,
 and deployment interaction in matching and history.`,
 	)
-	EnableDeploymentVersions = NewNamespaceBoolSetting(
+	EnableDeploymentVersions = NewGlobalBoolSetting(
```

Comment:
We should still use a per-ns dynamic config. In all other places (frontend/matching) the enabling is still per NS because we want to only allow private preview customers. The worker component does not need to use the per-ns config and should always return enabled though.

Reply:
ohh I misunderstood this my bad